### PR TITLE
webgpu: enlarge the splitted dimInner size

### PR DESCRIPTION
**Pull request by qjia7**
This PR enlarges the splitted dimInner size to reduce the number of calls to atomic operations.

Before
Conv2d 0.28ms input0: [5,5,2048] input1:[1,1,2048,32]
After
conv2d 0.12ms input0: [5,5,2048] input1:[1,1,2048,32]

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6755)

### CODE REVIEW

**Review by gyagp — APPROVED** [review]
LGTM

File: tfjs-backend-webgpu/src/matmul_packed_webgpu.ts [hunk 1]
```diff
@@ -209,8 +209,10 @@ export function makeMatMulPackedVec4Source(
     let batch = ${splitK ? '0' : 'i32(globalId.z)'};
     let globalRowStart = i32(workgroupId.y) * ${tileAOuter};
 
-    let numTiles = ${splitK ? '1' : '(uniforms.dimInner - 1) / TileInner + 1'};
-    var kStart = ${splitK ? 'i32(globalId.z) * TileInner' : '0'};
+    let numTiles = ${
+      splitK ? `${splitedDimInner / tileInner}` :
```
Comment (xhcao) [inline tfjs-backend-webgpu/src/matmul_packed_webgpu.ts:213]: Nit. Do we need to change it to `(splitedDimInner - 1) / TileInner + 1` if `splitedDimInner % tileInner` is not equal to 0

File: tfjs-backend-webgpu/src/matmul_packed_webgpu.ts [hunk 2]
```diff
@@ -323,8 +326,9 @@ export function makeMatMulPackedSource(
       let globalRowStart = i32(workgroupId.y) * ${tileAOuter};
 
       let numTiles = ${
-      splitK ? '1' : '(uniforms.dimInner - 1) / TileInner + 1'};
-      var kStart = ${splitK ? 'i32(globalId.z) * TileInner' : '0'};
+      splitK ? `${splitedDimInner / tileInner}` :
```
Comment (xhcao) [inline tfjs-backend-webgpu/src/matmul_packed_webgpu.ts:329]: Same here

**Review by xhcao — APPROVED** [review]

File: tfjs-backend-webgpu/src/matmul_packed_webgpu.ts (hunk 2 above)
Comment (qjia7) [reply tfjs-backend-webgpu/src/matmul_packed_webgpu.ts]: Change it to `Math.ceil(splitedDimInner / tileInner)`. Thanks.

**Pull request merged**